packages/react-components/package.json

```diff
@@ -493,6 +498,10 @@
     "./css/material/Shadow.css": "./css/material/Shadow.css",
     "./css/material/Typography.css": "./css/material/Typography.css",
     "./css/material/UserColors.css": "./css/material/UserColors.css",
+    "./utils/addOrUpdateEventListener.d.ts": "./utils/addOrUpdateEventListener.d.ts",
```

Comment:
These changes are removed when I run `npm install` or `npm run build` in the branch. Do you know why they are added?

Reply:
Oh, good catch! It turns out my local repo had some generated files from other branches that didn't get cleaned up by `npm install`. They were only removed after I ran `npm clean`.